### PR TITLE
Fix alphaem running

### PR DESCRIPTION
**Pull request by niclaurenti**
Implement APFEL running of alphaem in photon module: below tau mass alpha runs with nl=2, below muon mass nl=1 ecc.
Moreover below a given (arbitrary) threshold nf is set to zero.

### CODE REVIEW

**Comment by scarlehoff** [comment]
You need to wait for conda to pick up the new release of eko. We are still using conda.

**Comment by niclaurenti** [comment]
> You need to wait for conda to pick up the new release of eko. We are still using conda.

Yes I know, I just pushed it and when it will be updated in conda-forge I will rerun the tests

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:241]: Since this function is only used in this module and only as functional argument to `solve_inp` maybe it would be better to change it to `_rge` and instead of use `alpha` as the input (which could be confused with the class `Alpha` also defined in this module) use `alpha_ref` so that it is clear that it is just a value.

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:144]: As far as I can see this is the only place where rge is used I think.
Please add a comment before the return on why you select [0][-1] so that it is clear for future generations :P

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:102]: nl is missing.

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:1]: Please add a small docstr to this module (and also below to the class `Alpha`)

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:131]: same as above

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:115]: I think `self.betas_qed[(nf, nl)][1]` would not exist if QED is at LO. I think there should be some "if theory qed = 1" to return alpha_LO

File: validphys2/src/validphys/photon/alpha.py
Comment (niclaurenti) [reply validphys2/src/validphys/photon/alpha.py:241]: in this case `alpha` is not $\alpha_{ref}$ but $\alpha(t)$. I will think to a more clear name

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:156]:
```suggestion
        #  then the last element of the array which corresponds to alpha(q)
```

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:91]:
```suggestion
        Compute the running alphaem for nf fixed at, at least, NLO, using truncated method.
```

**Review by scarlehoff — COMMENTED** [review]
I guess this branch is what has been used for the latest fits to the ML right?

**Comment by niclaurenti** [comment]
> I guess this branch is what has been used for the latest fits to the ML right?

Yes!

**Comment by niclaurenti** [comment]
Moreover, until this PR is not merged some tests will be broken on master since it is downloading the new eko version

File: validphys2/src/validphys/photon/alpha.py
Comment (scarlehoff) [inline validphys2/src/validphys/photon/alpha.py:16]:
```suggestion
        self.qref = theory["Qref"]
```

For consistency

**Review by scarlehoff — APPROVED** [review]
I noticed... the stability of eko releases is yet another thing that should be addressed but anyway...

**Pull request merged**
